backend/amd64/emit.mlp

```diff
@@ -784,33 +784,21 @@ let emit_global_label s =
 let move (src : Reg.t) (dst : Reg.t) =
   if src.loc <> dst.loc then
     begin match src.typ, src.loc, dst.typ, dst.loc with
-    | (Float | Float32), Reg.Reg _, (Float | Float32), Reg.Reg _
-    | Vec128, _, Vec128, _ ->
-      (* Vec128 stack slots are always aligned. *)
+    | Float, Reg _, Float, Reg _
```

Comment:
(Was maybe easier to read with the or-pattern.)

Reply:
Yes, except it didn't catch float<->float32 moves!